packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts

```diff
@@ -15,23 +15,22 @@ const processEvent = async (response: Response) => {
     formattedHeaders[name.toLowerCase()] = value;
   });
   rawEvent = await response.text();
-  rawEvent = rawEvent.replace(/\s+/g, '');
   event = conversationCallbackWebhook.parseEvent(JSON.parse(rawEvent));
 };
 
 Given('the Conversation Webhooks handler is available', () => {
   conversationCallbackWebhook = new ConversationCallbackWebhooks(APP_SECRET);
 });
 
-Then('the Conversation event header contains a valid signature', () => {
-  assert.ok(conversationCallbackWebhook.validateAuthenticationHeader(formattedHeaders, rawEvent));
-});
-
 When('I send a request to trigger a "CAPABILITY" event', async () => {
   const response = await fetch('http://localhost:3014/webhooks/conversation/capability-lookup');
   await processEvent(response);
 });
 
+Then('the header of the Conversation event "CAPABILITY" contains a valid signature', () => {
```

Comment:
Is _just_ writing `Then('the header of the Conversation event {string} contains a valid signature'` could not help to have a single `Then` for all cases within this file ?

Reply:
On my side, no problem, but if you run the tests multithreaded, you'll need to know with header you are validating

Reply:
This is handled: https://github.com/sinch/sinch-sdk-java/pull/145/files#diff-39a2d8af5bc307b3fc35d03b23fa7639deae0d13f5c0411dba24c6c063ca600fR230
Receiving the `String e2eKeyword` parameter, java e2e tests are able to get event related to keyword

In current Node context: parameter won't be used but you could avoid code duplication

Reply:
Thanks for the information, I've removed the duplicated code on my side.